docs/source/basic/faq.rst

```diff
@@ -0,0 +1,80 @@
+
+Torch compile is a framework for generating optimized kernels. Its focus is on making pytorch code run faster by generating optimized kernels, with minimal code changes. Thunder is a framework for layering optimizations. It generates none of these optimizations itself, instead delegating to other libraries, including ``torch.compile`` and `nvfuser <https://github.com/NVIDIA/Fuser>`_. Our focus is on understandability, extendability, and usability.
+
+Modern deep learning optimization often involves stitching various kernels together, often from different sources. Thunder is designed to make it easy to use these tools together, and easy to add new tools to the mix.
+
+As such, the two are not necessarily comparable. Or, they are, but you would be comparing against a default configuration, which we expect you to extend and change anyway.
+
+
+
+============================================
+2. How can I use torch.compile with Thunder?
+============================================
+
+The correct way to use ``torch.compile`` with Thunder is to use the executor. This way, you get finer grained control over which parts of the model are handled by which executor.
+
+Calling ``torch.compile()`` and then ``thunder.jit()`` is not what you want. It doesn't give a good error message right now, but it should not work.
+
+Instead, register the executor like so::
+
+    import torch
+    import thunder
+
+    def model(x, y):
+        return x + y
+
+    exc_list = [thunder.extend.get_executor('torchcompile'), *thunder.get_always_executors()]
+    jmodel = thunder.jit(model, executors=exc_list)
+
+
+====================================================================================
+3. I have a CUDA, Triton, CUDNN, or other gpu kernel. How can I use it with thunder?
+====================================================================================
+
+Why, yes! You can register it as an operator for a custom executor. See :doc:`extending thunder <../intermediate/additional_executors>` for more information.
+
+
+========================================================================
+3. Do you support custom hardware, or accelerators that aren't Nvidia's?
+========================================================================
+
+Yes, executors are device-agnostic. The python executor, for example, runs the operation with cpython on the cpu. We've been focusing on Nvidia, but we welcome contributions for executors for other accelerator backends.
+
+
+=================================================================
+4. I ran ``thunder.jit(model)(*args)``, and my model didn't work!
+=================================================================
+
+Thunder is in alpha. There will be bugs, and many torch operations are not supported. Try to run ``thunder.examine(model)(*args)``. This will list the operations which are not supported, and if they are all supported, test the model for consistency against torch eager.
```

Comment:
```suggestion
Thunder is in alpha. There will be bugs, and many torch operations are not supported. Try to run ``from thunder.examine import examine; examine(model, *args)``. This will list the operations which are not supported, and if they are all supported, test the model for consistency against torch eager.
```